.github/workflows/js-checks.yml

```diff
@@ -15,23 +15,11 @@ concurrency:
 
 jobs:
   js-checks:
+    uses: hemilabs/actions/.github/workflows/js-checks.yml@main
+
+  validate-list:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: bloq/actions/setup-node-env@v1
-        with:
-          cache: npm
-      - run: npm run --if-present format:check
-      - run: npm run --if-present lint
-      - run: npm run --if-present deps:check
-  run-tests:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: bloq/actions/setup-node-env@v1
-        with:
-          cache: npm
-      - run: npm run test
-        env:
-          EVM_RPC_URL_43111: ${{ secrets.EVM_RPC_URL_43111 }}
-          EVM_RPC_URL_743111: ${{ secrets.WEB3_RPC_743111 }}
+      - uses: hemilabs/actions/setup-node-env@main
+      - run: npm run schema:validate
```

Comment:
If this step is added to i.e. the test script or as a pre-test, we could get rid of this job.

Reply:
Good idea ! Moved to a pretest script here [7c8ceb8](https://github.com/hemilabs/token-list/pull/30/commits/7c8ceb8369da45ba2089dca67c25d4643b0580a9)